### ISSUE TITLE
Add a max line length preference

### ISSUE THREAD

**Issue by thesamet** [scala-ide/scalariform#68]
**LOCKED TO AVOID +1s**: this feature would require a redesign and since scalariform is in maintenance mode, this is unlikely to happen unless we can get a new maintainer with time to do such large reworkings.

---

Some styleguides impose a maximal line length. Can you add a preference for such a constraint?

In come cases Scalariform joins multiple lines in the source files which can lead to long lines. One example would be:

```
implicit def canBuildFrom[B](implicit ordB: Ordering[B]):
  CanBuildFrom[KDTree[_], B, KDTree[B]] = new CanBuildFrom[KDTree[_], B, KDTree[B]] {
```

becomes a single long line:

```
implicit def canBuildFrom[B](implicit ordB: Ordering[B]):  CanBuildFrom[KDTree[_], B, KDTree[B]] = new CanBuildFrom[KDTree[_], B, KDTree[B]] {
```

**Comment by deepujain** [comment]
Its July 2015 and still need it :) :+1:

**Comment by gsmaverick** [comment]
+1

**Comment by mragab** [comment]
+1

**Comment by mohamedkomalo** [comment]
+1

**Comment by chellan** [comment]
+1

**Comment by speedcom** [comment]
+1

**Comment by ArtemPyanykh** [comment]
+1

**Comment by AThilenius** [comment]
+1

**Comment by mriehl** [comment]
+1

**Comment by bomgar** [comment]
+1

**Comment by MeiSign** [comment]
+1

**Comment by Ajk4** [comment]
+1

**Comment by godenji** [comment]
The long line example that in part motivates this request is fixed in 0.2.4

`implicit def canBuildFrom[B](implicit ordB: Ordering[B]):  CanBuildFrom[KDTree[_], B, KDTree[B]] = new CanBuildFrom[KDTree[_], B, KDTree[B]]`

Can be written as:
```
implicit def canBuildFrom[B]
  (implicit ordB: Ordering[B]):  CanBuildFrom[KDTree[_], B, KDTree[B]] = 
  new CanBuildFrom[KDTree[_], B, KDTree[B]] {

  }
```

For enforced line length take a look at the *scalafmt* library
